terraform/production/selection_rules.json

```diff
@@ -11,27 +11,72 @@
             "rule-action": "include",
             "filters": []
         },
+        {
+          "rule-type": "transformation",
+          "rule-id": "2",
+          "rule-name": "2",
+          "rule-action": "convert-lowercase",
+          "rule-target": "table",
+          "object-locator": {
+            "schema-name": "%",
+            "table-name": "DM_PERSONS"
+          }
+        },
         {
             "rule-type": "selection",
-            "rule-id": "2",
-            "rule-name": "2",
+            "rule-id": "3",
+            "rule-name": "3",
             "object-locator": {
                 "schema-name": "%",
                 "table-name": "DM_ADDRESSES"
             },
             "rule-action": "include",
             "filters": []
         },
+        {
+          "rule-type": "transformation",
+          "rule-id": "4",
+          "rule-name": "4",
+          "rule-action": "convert-lowercase",
```

Comment:
Does this definitely apply to the column name and not the data in the tables?

Reply:
Yeah, this shouldn't affect the data within the columns from what I understand.

Took inspiration from the "Example Convert to lowercase" AWS example [here](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.CustomizingTasks.TableMapping.html).

Reply:
nice :)